tsfile/src/main/java/org/apache/iotdb/tsfile/read/reader/page/PageReader.java

```diff
@@ -159,12 +162,25 @@ public void setFilter(Filter filter) {
     this.filter = filter;
   }
 
-  public void setDeletedAt(long deletedAt) {
-    this.deletedAt = deletedAt;
+  public void setDeleteIntervalList(List<TimeRange> list) {
+    this.deleteIntervalList = list;
   }
 
   @Override
   public boolean isModified() {
     return pageHeader.isModified();
   }
+
+  private boolean isDeleted(long timestamp, int deleteCursor) {
```

Comment:
Could you please recheck the logic? `deleteCursor` will not be changed after calling this method, since you didn't return it... When you call this method, `deleteCursor` keeps value 0.

Reply:
Fixed, thanks!